CMakeLists.txt

```diff
@@ -227,12 +227,7 @@ IF(LMMS_BUILD_LINUX AND WANT_VST)
 ENDIF()
 
 # Resolve qmake to full path for use in packaging scripts
-IF(WANT_QT6)
-	FIND_PROGRAM(QT_QMAKE_EXECUTABLE NAME qmake.exe
-		PATHS "${CMAKE_PREFIX_PATH}/bin" /opt/Qt6/bin) # TODO: add more paths here
-ELSE()
-	GET_TARGET_PROPERTY(QT_QMAKE_EXECUTABLE "${Qt5Core_QMAKE_EXECUTABLE}" IMPORTED_LOCATION)
-ENDIF()
+GET_TARGET_PROPERTY(QT_QMAKE_EXECUTABLE Qt${QT_VERSION_MAJOR}::qmake IMPORTED_LOCATION)
```

Comment:
I did have a doubt about this one line breaking something. Good that you caught it soon.

Reply:
I mean the one before what you changed.